build.cake

```diff
@@ -136,7 +136,20 @@ Task("TestNet45")
     .IsDependentOn("BuildNet45")
     .Does(() =>
     {
-        var testAssemblies = GetFiles("./tests/**/bin/" + configuration + "/*Tests.dll");
```

Comment:
We can't glob them anymore because we want to omit the Atlas ones here. Better to explicitly list which DLLs contain tests we want to run.